scm-ui/ui-components/src/repos/RepositoryAvatar.tsx

```diff
@@ -56,7 +56,7 @@ const renderExtensionPoint = (repository: Repository) => {
 };
 
 const RepositoryAvatar: FC<Props> = ({ repository }) => {
-  return <Avatar className="image is-64x64">{renderExtensionPoint(repository)}</Avatar>;
+  return <Avatar className="image is-48x48">{renderExtensionPoint(repository)}</Avatar>;
```

Comment:
I don't think that it is a good idea to change the default size of the repository avatar. We should make the size configurable and let the default untouched.

See https://github.com/scm-manager/scm-manager/pull/1738/commits/f65ca6ad0825baf62b478078aeee6d4e05c1ae33